packages/wallet-service/tests/integration.test.ts

```diff
@@ -248,274 +210,3 @@ test('load wallet, and simulate DLQ event', async () => {
     expect.any(Logger),
   );
 }, 60000);
-
-// eslint-disable-next-line jest/prefer-expect-assertions, jest/expect-expect
-test('start wallet and then receive blocks and txs', async () => {
```

Comment:
We don't have tests for this, need to create tests that use both the lambdas and the daemon